chart/ingress-operator/values.yaml

```diff
@@ -0,0 +1,16 @@
+# https://github.com/openfaas-incubator/ingress-operator
+image: openfaas/ingress-operator:0.6.2
+
+replicas: 1
+
+create: false
```

Comment:
Won't we want to get rid of this value as this is explicitly True when installing this now